### ISSUE TITLE
cljs repl stop working after stop then start autobuild

### ISSUE THREAD

**Issue by lucywang000** [thheller/shadow-cljs#700]
To reproduce:

1. in a `yarn shadow-cljs clj-repl`, run `(shadow/watch-set-autobuild! :app false)`, then followed by `(shadow/watch-set-autobuild! :app true)`
2. now the cljs repl won't work anymore:
```
missing cljs.user, repl not properly configured (must have analyzed cljs.user by now)                                                                       (1/8 results) [30/121]
{}
ExceptionInfo: missing cljs.user, repl not properly configured (must have analyzed cljs.user by now)
```

could be fixed by commenting out one lines:
```
--- a/src/main/shadow/cljs/devtools/server/worker/impl.clj
+++ b/src/main/shadow/cljs/devtools/server/worker/impl.clj
@@ -772,8 +772,8 @@
     ;; compile immediately, autobuild is then checked later
     (-> worker-state
         (assoc :autobuild true)
-        (build-configure)
+        #_(build-configure)
        (build-compile)
         )))
```

**Comment by thheller** [comment]
This should be fixed as of 2.8.110.

**Comment by thheller** [comment]
There is one more issue left when using a custom `:repl-init-ns` that might lead to the `cljs.user` error above. That will be taken care of in the next release.
